examples/cosmos/ics20/denomTrace.qnt

```diff
@@ -47,17 +45,17 @@ module denomTrace {
     }
   }
 
-  /// `isProperChannelSetup` is true iff channel names are unique by chain
-  pure def isProperChannelSetup(setup: str -> str -> str): bool =
-    pure val chainNames = Set("A", "B", "C") // should be setup.keys(), but that's breaking the effect checker somehow (see issue #808)
+  /// `isProperChannelTopology` is true iff channel names are unique by chain
+  pure def isProperChannelTopology(topology: ChannelTopology): bool =
```

Comment:
If this is actually the only test, I'd prefer to name the predicate according to what it determines, e.g.,

```suggestion
  /// `channelNamesUniquePerChain` is true iff channel names are unique by chain
  pure def channelNamesUniquePerChain(topology: ChannelTopology): bool =
```

This way, I don't have to think what "proper" means when I see a call to `isProperChannelTopology`.